graylog2-server/src/main/java/org/graylog2/users/UserImpl.java

```diff
@@ -133,7 +141,9 @@ public boolean isReadOnly() {
                 .put(USERNAME, new LimitedStringValidator(1, MAX_USERNAME_LENGTH))
                 .put(PASSWORD, new FilledStringValidator())
                 .put(EMAIL, new LimitedStringValidator(1, MAX_EMAIL_LENGTH))
-                .put(FULL_NAME, new LimitedOptionalStringValidator(MAX_FULL_NAME_LENGTH))
+                .put(FIRST_NAME, new LimitedOptionalStringValidator(MAX_NAME_LENGTH))
+                .put(LAST_NAME, new LimitedOptionalStringValidator(MAX_NAME_LENGTH))
+                .put(FULL_NAME, new LimitedOptionalStringValidator(MAX_NAME_LENGTH))
```

Comment:
Shouldn't the full name length be validated with `MAX_NAME_LENGTH * 2 + 1`?

Reply:
Good catch. Perhaps it makes sense to use `100` for the first/last and keep `200` for the full, since that was what was used before. I've changed this.